homeassistant/components/subaru/sensor.py

```diff
@@ -128,7 +131,7 @@
     },
     {
         SENSOR_TYPE: "EV Time to Full Charge",
-        SENSOR_CLASS: None,
+        SENSOR_CLASS: DEVICE_CLASS_TIMESTAMP,
         SENSOR_FIELD: sc.EV_TIME_TO_FULLY_CHARGED,
         SENSOR_UNITS: TIME_MINUTES,
```

Comment:
I noticed now that we didn't change the unit or how the entity state is calculated for this entity type. The state should be an absolute UTC isoformatted timestamp. The unit should be `None`.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes